src/screen/Screen.java

```diff
@@ -97,6 +105,11 @@ public Menu run() {
 	protected void update() {
 	}
 
```

Comment:
I think `draw` method is same on all child of `Screen`.
So, `draw` method should be created in `Screen` and call in `update` method.
And, All child should be override update with `super.update()` call.

ex)
```java
// GameSettingScreen.java

@Override
protected final update() {
    super.update();
    ...
}

// Screen.java
protected void update() {
    draw();
}

protected void draw() {
    renderer.initDrawing();
    renderer.drawEntities(entities);
    renderer.completeDrawing();
}
```